### PR TITLE
update to capistrano 3

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: deployment/tasks/robots.cap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace :orchestra do
+    desc "Generate sitemap.xml files"
+    task :generate_robots do
+        on roles(:app) do
+            execute :php, "#{fetch(:release_path)}/app/console orchestra:robots:generate --env=prod "
+        end
+    end
+    after 'deploy:finishing', 'orchestra:generate_robots'
+end
```
Comment (itkg-ngilain) [inline deployment/tasks/robots.cap]: NL@EOF

File: deployment/tasks/sitemap.cap [hunk 2]
```diff
@@ -0,0 +1,9 @@
+namespace :orchestra do
+    desc "Generate sitemap.xml files"
+    task :generate_sitemaps do
+        on roles(:app) do
+            execute :php, "#{fetch(:release_path)}/app/console orchestra:sitemaps:generate --env=prod "
+        end
+    end
+    after 'deploy:finishing', 'orchestra:generate_sitemaps'
+end
```
Comment (itkg-ngilain) [inline deployment/tasks/sitemap.cap]: NL@EOF

**Comment by nicolasThal** [comment]
@itkg-ngilain pr updated

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**